composer.json

```diff
@@ -3,6 +3,7 @@
     "type": "library",
     "require": {
         "php": ">=5.6",
+        "guzzlehttp/guzzle": "~6.0",
```

Comment:
Is this going to cause any issues with customers which are not using this version of Guzzle, or, using the _really_ old version?

Reply:
Also, should we bind to `~6.0.0` to only get bug fixes, or, are you happy with potentially getting features also?

Reply:
Nope as meteor is a self container package

Reply:
👍

Reply:
Done